torch/csrc/jit/backends/backend_init.cpp

```diff
@@ -329,13 +349,6 @@ void initJitBackendBindings(PyObject* module) {
           method_ct.format(method_te), loweredModuleResolver());
     }
 
-    // Run preprocess so that __processed_module is set correctly before
-    // compilation.
-    loweredModule.run_method(
```

Comment:
I'm wondering how is it backward compatible without running __preprocess, when there's no BackendPreprocessFunction? In such a case, would this piece of code still be needed?